simulator.code-workspace

```diff
@@ -64,9 +68,9 @@
 				"name": "Debug LVGL demo with gdb",
 				"type": "cppdbg",
 				"request": "launch",
-			        "program": "${workspaceFolder}/build/bin/demo",
+			        "program": "bin/main",
```

Comment:
I have to change this line as below to let it work for me:
"program": "${workspaceFolder}/bin/main",